libs/core/src/components.d.ts

```diff
@@ -2064,6 +2116,51 @@ declare namespace LocalJSX {
          */
         "variant": 'primary' | 'availability' | 'filter';
     }
+    interface PdsText {
+        /**
+          * Sets the text alignment.
+         */
+        "align"?: 'start' | 'center' | 'end' | 'justify';
+        /**
+          * Sets the text color.
+         */
+        "color"?: | 'primary'
```

Comment:
Question: why only allow access to semantic? Will we not allow the ability to add any color token available?

Reply:
@QuintonJason Per discussion, we have alignment with design for our semantic text colors. If we need to branch out and allow colors past those, we can create a follow-up.